src/main/resources/findologic_20.xsd

```diff
@@ -304,7 +295,21 @@
                 <xs:complexType>
                     <xs:simpleContent>
                         <xs:extension base="xs:dateTime">
-                            <xs:attribute type="usergroupHash" name="usergroup" use="optional"/>
+                            <xs:attribute type="nonEmptyString" name="usergroup" use="optional"/>
+                        </xs:extension>
+                    </xs:simpleContent>
+                </xs:complexType>
+            </xs:element>
+        </xs:sequence>
+    </xs:complexType>
+
+    <xs:complexType name="overriddenPricesType">
```

Comment:
:thinking: This is largely identical to the `pricesType`, except for `minOccurs`. I assume you didn't choose to do a type extension, because it was equally verbose to defining a new type that's almost the same.

Reply:
In the follow-up the `minOccurs` changes to the same value as for price. I've added this in a comment of the PR so we can handle it there
https://github.com/findologic/xml-export/pull/17/files#r996861486